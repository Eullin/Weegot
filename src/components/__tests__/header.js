import React from "react"
import { render } from "@testing-library/react"
import Header from '../header'

test("Displays the correct title", () => {
  const { getByTestId } = render(<Header />)
  // Assertion
  expect(getByTestId("header")).toMatchSnapshot()
  // --> Test will pass
})