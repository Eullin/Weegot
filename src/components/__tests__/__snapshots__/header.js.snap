// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays the correct title 1`] = `
<header
  data-testid="header"
  style="background: rebeccapurple; margin-bottom: 1.45rem;"
>
  <div
    style="margin: 0px auto; max-width: 960px; padding: 1.45rem 1.0875rem;"
  >
    <h1
      style="margin: 0px;"
    >
      <a
        aria-current="page"
        class=""
        href="/"
        style="color: white; text-decoration: none;"
      />
    </h1>
  </div>
</header>
`;
